flow/packet-headers/snmpv2c.yaml

```diff
@@ -0,0 +1,366 @@
+              x-field-uid: 8
+            big_counter_value:
+              x-field-uid: 9
+            unsigned_integer_value:
+              x-field-uid: 10
+        integer_value:
+          x-field-pattern:
+            description: >-
+                Integer value returned for the requested OID.
+            format: integer
+            length: 32
+            default: 0
+            signed: true
+            features: [count]
+          x-field-uid: 2
+        string_value:
+          description: >-
+            It contains the hex bytes of the value to be sent. 
+            As of now it is restricted to 5000 bytes.
+          type: string
+          maxLength: 10000
+          default: "string"
+          x-field-uid: 3
+        object_identifier_value:
+          x-include: '#/components/schemas/Flow.Snmpv2c.Common/properties/object_identifier'
+          x-field-uid: 4
+        ip_address_value:
+          $ref: '#/components/schemas/Flow.Snmpv2c.VariableBindingIPValue'
+          x-field-uid: 5
+        counter_value:
+          x-field-pattern:
+            description: >-
+              Counter returned for the requested OID.
+            format: integer
+            length: 32
+            default: 0
+            features: [count]
+          x-field-uid: 6
+        timeticks_value:
+          x-field-pattern:
+            description: >-
+              Timeticks returned for the requested OID.
+            format: integer
+            length: 32
+            default: 0
+            features: [count]
+          x-field-uid: 7
+        arbitrary_value:
+          description: >-
+            It contains the hex bytes of the value to be sent. 
+            As of now it is restricted to 5000 bytes.
+          type: string
+          format: hex
+          maxLength: 10000
+          default: "00"
+          x-field-uid: 8
+        big_counter_value:
+          x-field-pattern:
+            description: >-
+              Big counter returned for the requested OID.
+            format: integer
+            length: 64
+            default: 0
+            features: [count]
+          x-field-uid: 9
+        unsigned_integer_value:
+          x-field-pattern:
+            description: >-
+              Unsigned integer value returned for the requested OID.
+            format: integer
+            length: 32
+            default: 0
+            features: [count]
+          x-field-uid: 10
+    Flow.Snmpv2c.VariableBindingIPValue:
+      type: object
+      properties:
+        choice:
+          type: string
+          default: ipv4
+          x-field-uid: 1
+          x-enum:
+            ipv4:
+              x-field-uid: 1
+            ipv6:
+              x-field-uid: 2
+        ipv4:
+          x-field-pattern:
+            description: >-
+              IPv4 address returned for the requested OID.
+            format: ipv4
+            default: 0.0.0.0
+            features: [count]
+          x-field-uid: 2
+        ipv6:
```

Comment:
According to [SNMPv2-RFC3416](https://www.ietf.org/rfc/rfc3416.txt?number=3416) supported `ApplicationSyntax` are

```
   ApplicationSyntax ::= CHOICE {
         ipAddress-value        IpAddress,
         counter-value          Counter32,
         timeticks-value        TimeTicks,
         arbitrary-value        Opaque,
         big-counter-value      Counter64,
         unsigned-integer-value Unsigned32 }
```
SNMP Data Type is supporting single IP type (`IpAddress`) with `SIZE (4)`
```
IpAddress ::= [APPLICATION 0] IMPLICIT OCTET STRING (SIZE (4))
```
So, I am suspecting RFC talked about for IPv4 address NOT IPv6 address. Because if this is IPv6 then what is the length of it.

Could you please double check it otherwise we have to remove IPv6.